ample/util/pdb_edit.py

```diff
@@ -1382,12 +1379,12 @@ def renumber_residues_gaps(pdbin, pdbout, gaps, start=1):
 def Xselect_residues(inpath=None, outpath=None, residues=None):
```

Comment:
maybe remove all the `X<something>` functions? We clearly don't use them anymore

Reply:
Good idea. Done